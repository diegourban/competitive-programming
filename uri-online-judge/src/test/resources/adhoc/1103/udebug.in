23 30 0 34
23 27 1 59
22 30 8 30
21 00 8 00
1 5 3 5
23 59 0 34
21 33 21 10
23 59 1 27
0 0 0 0