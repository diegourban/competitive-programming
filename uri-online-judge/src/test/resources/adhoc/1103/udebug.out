64
152
600
660
120
35
1417
88
